docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,31 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
```

Comment:
hm, slightly confused by the `key1=value1,key2=value2`; to set multiple labels, I have to use multiple `--label-add` flags, not a comma-separated list, or am I mistake ?